src/Client.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Openpix\PhpSdk;
+
+use Openpix\PhpSdk\Resources\Costumers;
+
+class Client
+{
+    public function costumers()
```

Comment:
customers

Reply:
I'll fix it by resolving the issue #3 in a new pull request :pray: